certification/internal/engine/engine.go

```diff
@@ -224,6 +227,13 @@ func (c *CraneEngine) ExecuteChecks(ctx context.Context) error {
 	return nil
 }
 
+func appendUnlessOptional(results []runtime.Result, result runtime.Result) []runtime.Result {
+	if result.Check.Metadata().Level == "optional" {
```

Comment:
I know I already gave you the LGTM, but:
1) Should this be an exported constant, usable by checks?
2) Should we add dev docs?

Reply:
Now that we have more then one check implementing `optional`, I agree on both points above

Reply:
Neither are bad ideas. (I thought we already had dev docs? Or, since this would be an exported constant, it will end up in API docs)

Reply:
That works for me.